### ISSUE TITLE
i18n: Advertising Page Redirecting to My Home

### ISSUE THREAD

**Issue by kriskorn** [Automattic/wp-calypso#71858]
### Quick summary

When a non-English language is selected in https://wordpress.com/me/account then the Tools -> Advertising link redirects to My Home and does not open. 

https://user-images.githubusercontent.com/12064241/211570235-9e3565df-9c3d-45bb-8dda-d39e33c51f09.mp4

### Steps to reproduce

1. Go to https://wordpress.com/me/account 
2. Under **Interface Language** choose a non-English language.
3. Go to My Sites -> Tools and click on Advertising.
4. You will be redirected to My Home instead of opening the Advertising page.

### What you expected to happen

~~To see the Advertising page to promote posts.~~
This functionaliy is not yet available for non-English customers, so the **Advertising** link should not be shown in the menu for non-English languages. 

### What actually happened

~~The link redirects to My Home instead.~~
This functionaliy is not yet available for non-English customers, so the **Advertising** link should not be shown in the menu for non-English languages. 

### Context

* 38433521-hc
* p1673358008090789-slack-dotcom-escalations

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

~~Change **Interface Language** to English under Account Settings.~~ 
This functionaliy is not yet available for non-English customers, so the **Advertising** link should not be shown in the menu for non-English languages.

**Comment by gmovr** [comment]
` userData[ 'has_promote_widget' ]` in `usePromoteWidget` hook returns `false` for non-EN locales, and true for English. This causes a redirect in `main.tsx` where the PromoteWidgetStatus is checked, and the user is redirected if it's `DISABLED` 

```javascript
if ( usePromoteWidget() === PromoteWidgetStatus.DISABLED ) {
		page( '/' );
	}
	//...
```

**Comment by RafaelFunchal** [comment]
The issue has been fixed by #71880
